src/tools/compiletest/src/main.rs

```diff
@@ -1,5 +1,5 @@
 #![crate_name = "compiletest"]
-#![deny(warnings)]
+#![feature(vec_remove_item)]
```

Comment:
Huh. This is surprising to me -- also see the comment just below that says "don't do anything other than feature(test)" -- could we instead replace the bit which is calling remove_item (similar to other cleanups).

Reply:
Oh, sorry! That was just a merge mistake on my part (a conflict with #73539).